rate-estimation/include/PreColumn.C

```diff
@@ -100,13 +103,16 @@ bool PreColumn::InsertInMenu(std::string L1name, bool value, float pu, bool rewe
 
   mL1Seed[L1name].eventfire = false;
   if (value)
-    mL1Seed[L1name].ncounts+=ev_puweight;
+    mL1Seed[L1name].ncounts += ev_puweight * prescale_precision;
 
   if (mL1Seed[L1name].prescale == 0)
     return false;
 
-  if ( int(mL1Seed[L1name].ncounts) % mL1Seed[L1name].prescale == 0) 
+  if ( mL1Seed[L1name].ncounts >= mL1Seed[L1name].prescale_discret)
+  {
```

Comment:
This code block is executed independently of `value` which becomes relevant in case of overshooting due to high `ev_puweight`. -> `mL1Seed[L1name].ncounts` is reduced to a value below the prescale.
Possible solutions:
- only reduce `mL1Seed[L1name].ncounts` if the event fulfills the trigger seed conditions (and the counter is higher than the prescale)
- reduce to a value in `[0, prescale_discrete]` immediately when exceeding the prescale.

Reply:
I just decided for the first option, to have the code in a state that is not obviously wrong, even though reweighting is not used currently.